css/components/button.css

```diff
@@ -0,0 +1,27 @@
+.btn{
+  padding: 0.8rem 1rem;
+  border: none;
+  border-radius: 0.5rem;
+  
+}
+
+.btn:hover{
+  opacity: 0.5;
```

Comment:
Small suggestion,  increase the opacity closer to 1, this kinda makes it look like the button is disabled when I hover it.
You can also add transition property to .btn {transition: all 0.3s ease 0s;}.
In my opinion, it makes it a bit smoother when someone hovers over.

Reply:
done